app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h1>post pages</h1>
\ No newline at end of file
+<h1>post pages</h1>
+<div class="posts_container">
+  <div class="user_posts">
+     <img src="<%= @user.photo %>" alt="user_pic" />
+     <div>
+        <p><%= @user.name %></p>
+        <p class="posts_count">Number of posts: <%= @user.posts_counter || 0 %></p>
+     </div>
+  </div>
+  <div>
+    <% @posts.each do |post| %>
+      <% if @posts.empty? %>
+        <p>User doesn't have posts yet</p>
+      <% else %>
+        <div>
+            <h2><%= post.title %></h2>
+            <p><%= post.text %></p>
+            <ul>
+              <li>Comment: <%= post.comments.count%></li>
+              <li>Likes: <%= post.likes.count%></li>
+            </ul>
+        </div>
+        <div>
+          <% if post.most_recent_comments.empty? %>
+              <p>No comment for this post</p>
+          <% else %>
+            <% post.most_recent_comments.each do |comment| %>
+              <%= comment.author.name %>  :<%= comment.text %>
+            <% end %>
+          <% end %>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+</div>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169436457-af6af1ad-90ec-4e39-aad1-c38769f37762.png)


- [ ] Kindly note that you need to follow the wireframes that were provided in the requirements, I know that the style does not matter but you need to add some basic CSS to follow the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md). check the wireframes :point_down: 

![image](https://user-images.githubusercontent.com/65068771/169435586-73bdb804-e322-44fe-a337-6f7ca8c310d0.png)

- [ ] Kindly also make sure when I click on the post, it redirects me to the `post#show` view page.